src/startProfil.js

```diff
@@ -0,0 +1,27 @@
+const { getConfig } = require('./config');
+
+let profils;
+let profilNames;
+const init = () => {
+  const sortByAppName = (a, b) => {
+    if (a.name > b.name) return 1;
+    if (a.name < b.name) return -1;
+    return 0;
+  };
+
+  profils = (getConfig().profils || []).sort(sortByAppName);
```

Comment:
This seems copy-pasted, at the moment I see no reason for having to sort the profiles. 🤔